### PR TITLE
fixes #5444 - add param_false? host method and allow in templates

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by lzap** [comment]
Good catch with the missing param_true? method, I had to test it with safemode turned off. Thanks. +1

File: app/models/host/managed.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ class Jail < ::Safemode::Jail
     allow :name, :diskLayout, :puppetmaster, :puppet_ca_server, :operatingsystem, :os, :environment, :ptable, :hostgroup, :location,
       :organization, :url_for_boot, :params, :info, :hostgroup, :compute_resource, :domain, :ip, :mac, :shortname, :architecture,
       :model, :certname, :capabilities, :provider, :subnet, :token, :location, :organization, :provision_method,
-      :image_build?, :pxe_build?, :otp, :realm
+      :image_build?, :pxe_build?, :otp, :realm, :param_true?, :param_false?
```
Comment (domcleal) [inline app/models/host/managed.rb:51]: Could you add these to Hostgroup too?  host_common is used in both

Comment (stbenjam) [inline app/models/host/managed.rb:51]: Yup, done!

**Comment by domcleal** [comment]
Thanks @stbenjam, merged as 6807d42 for Foreman 1.5.0.

**Pull request closed without merging**